### PR TITLE
Specify namespace when creating pod

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
specify the namespace (pool) when creating a new pod

### CODE REVIEW

**Review by jdpgrailsdev — APPROVED** [review]
:shipit:

**Review by git-phu — COMMENTED** [review]
I believe what we need is to specify node selectors when creating the orchestrator pod, which is different from specifying the kubernetes namespace

**Comment by git-phu** [comment]
in KubePodProcess we are already setting tolerations and selectors here https://github.com/airbytehq/airbyte/blob/ebbe5a9a18a59973531d81d3edaf62db734ed323/airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java#L510-L512

So I believe what is happening is that in the container orchestrator app we are setting the tolerations and nodeSelectors properly when calling the KubePodProcess constructor, but when creating the actual orchestrator pods (which I believe should happen in the worker app) we are not setting these configs.

File: airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java [hunk 1]
```diff
@@ -352,11 +353,13 @@ public void create(final Map<String, String> allLabels,
         .withContainers(mainContainer)
         .withInitContainers(initContainer)
         .withVolumes(volumes)
+        .withNodeSelector(nodeSelectors)
```
Comment (git-phu) [inline airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java:356]: I think we should include tolerations as well like in `KubePodProcess` https://github.com/airbytehq/airbyte/blob/ebbe5a9a18a59973531d81d3edaf62db734ed323/airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java#L510

However, our current node pools don't actually use taints, so we don't need tolerations for this to work for our own cluster.  In general though, we should include it (maybe in a separate followup PR. `buildPodTolerations` looks like a private method though it should be easy to extract out of `KubePodProcess`.)

**Review by git-phu — APPROVED** [review]
looks good, thanks!

I think we should support pod tolerations as well for completeness, but for our own cluster setup we don't need it so I'll approve this PR as is so we can get it deployed.

We should add pod toleration support though, if not in this PR then in a follow up.

File: airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java [hunk 2]
```diff
@@ -352,11 +353,13 @@ public void create(final Map<String, String> allLabels,
...
         .endSpec()
         .build();
 
     // should only create after the kubernetes API creates the pod
     final var createdPod = kubernetesClient.pods()
+
```
Comment (pmossman) [inline airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java:362]: nit: extra newline

**Review by pmossman — APPROVED** [review]
Thanks for taking care of this!

**Comment by benmoriceau** [comment]
> looks good, thanks!
> 
> I think we should support pod tolerations as well for completeness, but for our own cluster setup we don't need it so I'll approve this PR as is so we can get it deployed.
> 
> We should add pod toleration support though, if not in this PR then in a follow up.

Ok, I'll do it in a follow up.

**Comment by benmoriceau** [comment]
the build seems to fail because of: https://airbytehq-team.slack.com/archives/C046KQF5GBT/p1668531356052829

**Pull request merged**